partiql-planner/src/main/resources/builtins.sql

```diff
@@ -1,1681 +0,0 @@
--- [trim] ---------
```

Comment:
Just curious why are these *.sql files getting deleted?

Reply:
They're inconsequential, outdated, and shouldn't have been pushed to begin with.